public/app/plugins/datasource/elasticsearch/configuration/DataLinks.test.tsx

```diff
@@ -1,84 +1,63 @@
-import { mount } from 'enzyme';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
 import React from 'react';
-import { act } from 'react-dom/test-utils';
 
-import { Button } from '@grafana/ui';
+import { DataLinkConfig } from '../types';
 
-import { DataLink } from './DataLink';
-import { DataLinks } from './DataLinks';
+import { DataLinks, Props } from './DataLinks';
 
-describe('DataLinks', () => {
-  let originalGetSelection: typeof window.getSelection;
-  beforeAll(() => {
-    originalGetSelection = window.getSelection;
-    window.getSelection = () => null;
-  });
+const setup = (propOverrides?: Partial<Props>) => {
+  const props: Props = {
+    value: [],
+    onChange: jest.fn(),
+    ...propOverrides,
+  };
 
-  afterAll(() => {
-    window.getSelection = originalGetSelection;
-  });
+  Object.assign(props);
```

Comment:
this `Object.assign` doesn't do anything anymore so can be removed 👍

Reply:
Done :)